components/comic/RawComicItem.tsx

```diff
@@ -1,24 +1,36 @@
-import Box, { BoxProps } from '@mui/material/Box'
+import { BoxProps } from '@mui/material/Box'
 import { RawComic } from '@/models/comic/rawComic'
 import { RoutePath } from 'enums/routePath'
 import SkeletonImage from '../SkeletonImage'
-import Link from 'next/link'
+import { useRouter } from 'next/navigation'
 import clsx from 'clsx'
-
 interface Props extends BoxProps {
 	comic: RawComic
 }
```

Comment:
This is incorrect. The base element of this component is `tr` but you're passing it `BoxProps` which is by default a `div` element. This is type unsafe and type misleading.

If you wish to know how to properly handle this component and it's props, we can jump on a quick call. No pressure, we don't have to do it